src/longwave_radiation.jl

```diff
@@ -0,0 +1,203 @@
+"""
+    function radset!(model::PrimitiveEquationModel) where {NF<:AbstractFloat}
+        
+Compute energy fractions in four longwave bands as a function of temperature.
+
+"""
+function radset!(model::PrimitiveEquationModel{NF}) where {NF<:AbstractFloat}
+    @unpack epslw, fband = model.parameters
+
+    nband = size(fband)[2]
+    @assert nband == 4 "Only four bands are supported, given $nband"
+
+    eps1 = 1 - epslw
+    for jtemp = 200:320
+        fband[jtemp, 2] = (NF(0.148) - NF(3.0e-6) * (jtemp - 247)^2) * eps1
+        fband[jtemp, 3] = (NF(0.356) - NF(5.2e-6) * (jtemp - 282)^2) * eps1
+        fband[jtemp, 4] = (NF(0.314) + NF(1.0e-5) * (jtemp - 315)^2) * eps1
+        fband[jtemp, 1] = eps1 - (fband[jtemp, 2] + fband[jtemp, 3] + fband[jtemp, 4])
+    end
+    for jb = 1:4
+        for jtemp = 100:199
+            fband[jtemp, jb] = fband[200, jb]
+        end
+        for jtemp = 321:400
+            fband[jtemp, jb] = fband[320, jb]
+        end
+    end
+    return fband
+end
```

Comment:
Do I understand this right that `radset!` is a function that is only executed once as a precomputing step? Cause then I would suggest to add a `ParameterizationConstants` field to `PrimitiveEquationModel` and execute `radset!` in `initialize_speedy`.

Reply:
(and move `fband` from `Parameters` into `ParameterizationConstants`, cause it's not a parameter a user would choose but a precomputed array with constant entries ...)